numpy/polynomial/_polybase.py

```diff
@@ -273,6 +277,74 @@ def __str__(self):
         name = self.nickname
         return format % (name, coef)
 
+    @classmethod
+    def _repr_latex_term(cls, i, arg_str, needs_parens):
+        if cls.basis_name is None:
+            raise NotImplementedError(
+                "Subclasses must define either a basis name, or override "
+                "_repr_latex_term(i, arg_str, needs_parens)")
+        # since we always add parens, we don't care if the expression needs them
+        return "{{{basis}}}_{{{i}}}({arg_str})".format(
+            basis=cls.basis_name, i=i, arg_str=arg_str
+        )
+
+    def _repr_latex_(self):
+        # get the scaled argument string to the basis functions
+        off, scale = self.mapparms()
+        if off == 0 and scale == 1:
+            term = 'x'
+            needs_parens = False
+        elif scale == 1:
+            term = '{} + x'.format(off)
+            needs_parens = True
+        elif off == 0:
+            term = '{}x'.format(scale)
+            needs_parens = True
+        else:
+            term = '{} + {}x'.format(off, scale)
+            needs_parens = True
+
+        # filter out uninteresting coefficients
+        filtered_coeffs = [
+            (i, c)
+            for i, c in enumerate(self.coef)
+            # if not (c == 0)  # handle NaN
+        ]
+
+        mute = r"\color{{LightGray}}{{{}}}".format
+
+        parts = []
+        for i, c in enumerate(self.coef):
+            # prevent duplication of + and - signs
+            if i == 0:
+                coef_str = '{}'.format(c)
```

Comment:
Unfortunately `'{}'.format(some_float)` doesn't generate valid latex for coefficients using scientific notation. I'm not sure what the best thing to do here is - do we show the coefficients as monospaced and keep the `e`, to make it easy to copy/paste them?

Reply:
> do we show the coefficients as monospaced and keep the e, to make it easy to copy/paste them?
Makes sense, at least in the near term. This might be one of the things that garners feedback. We should probably decide how to set various options for the display, but that can wait.
> Do you think we should gain most of the feedback before or after merging?
I was thinking after merge. I don't think it matters much if we change things later, and we won't know what people want in practice until we have people using it :) Maybe mark it as experimental in the release notes. Can make it a highlight just so folks see it.

Reply:
Yes, this part of latex formatting is annoying; we have a [specific method](https://github.com/astropy/astropy/blob/master/astropy/units/format/latex.py#L88) that does it for units/quantities